swagger-templates/php/README.mustache

```diff
@@ -114,12 +114,51 @@ Class | Method | HTTP request | Description
 
 {{/authMethods}}
 
+## Pagination of V1 Endpoints
+
+V1 Endpoints return pagination information via http headers. In order to obtain
+response headers and extract the `batch_token` parameter you will need to follow
+the following steps:
+
+1. Use the full information endpoint methods of each API to get the response HTTP
+Headers. They are named as their simple counterpart with a `WithHttpInfo` suffix.
+Hence `listEmployeeRoles` would be called `listEmployeeRolesWithHttpInfo`. This
+method returns an array with 3 parameters: `$response`, `$http_status`, and
+`$http_headers`.
+2. Use `$batch_token = \SquareConnect\ApiClient::getV1BatchTokenFromHeaders($http_headers)`
```

Comment:
needs to add the implementation to the template, so it could stay in ApiClient. (not overwritten by later auto generated code.)